app/src/androidTest/resources/all.json

```diff
@@ -0,0 +1,3606 @@
+{
```

Comment:
Don't we need all sessions? It's enough to use at most 10 sessions for testing, isn't it?